src/styles/_variables.scss

```diff
@@ -0,0 +1,9 @@
+$color-blue: #00acdc;
+$color-dark: #060b35;
+$color-grey: #616070;
+$font-weight-normal: 400;
+$font-weight-medium: 500;
+$font-weight-bold: 700;
```

Comment:
That's redundant. I've had the same problem in my task. You can use normal for 400, bold for 700, and set 500 explicitly for your fonts

https://developer.mozilla.org/ru/docs/Web/CSS/font-weight

Reply:
Not sure abouth that, though